hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/TestHoodieClientOnCopyOnWriteStorage.java

```diff
@@ -546,7 +571,10 @@ private void testUpsertsUpdatePartitionPath(IndexType indexType, HoodieWriteConf
     HoodieTableMetaClient.initTableType(metaClient.getHadoopConf(), metaClient.getBasePath(),
         metaClient.getTableType(), metaClient.getTableConfig().getTableName(), metaClient.getArchivePath(),
         metaClient.getTableConfig().getPayloadClass(), VERSION_0);
-    SparkRDDWriteClient client = getHoodieWriteClient(hoodieWriteConfig, false);
+    // Set rollback to LAZY so no inflights are deleted
+    hoodieWriteConfig.getProps().put(HoodieCompactionConfig.FAILED_WRITES_CLEANER_POLICY_PROP,
```

Comment:
We need to do something about the repeated line

`getConfig().getProps().put(HoodieCompactionConfig.FAILED_WRITES_CLEANER_POLICY_PROP,
        HoodieCleaningPolicy.HoodieFailedWritesCleaningPolicy.LAZY.name());`

even if you want to override only selectively. Can we add an override for `getConfig(boolean)` which decides whether or not we do lazy cleaning of failed writes.